src/components/MatchCard/MenuButton.tsx

```diff
@@ -58,8 +72,16 @@ export const MenuButton = ({ id }: { id: number }) => {
         >
           Edit
         </MenuItem>
-        <MenuItem onClick={handleClose}>Delete</MenuItem>
+        <MenuItem onClick={() => setIsDeleteDialogOpen(!isDeleteDialogOpen)}>
+          Delete
+        </MenuItem>
       </Menu>
+      <DeleteDialog
+        isDeleteDialogOpen={isDeleteDialogOpen}
```

Comment:
You can remove `delete` from all those prop names. The context is already set by the component name